### PR TITLE
Feat/several validators

### PR DESCRIPTION
**Pull request by Gamuza469**
Implement several new validators and improve existing validators and tests.

### CODE REVIEW

File: src/EmptyValidator.js [hunk 1]
```diff
@@ -1,14 +1,19 @@
...
+        }
+
         super();
-        this.withErrorMessage('Must not be empty');
+        this.withErrorMessage('Value must be empty.');
     }
 
     isValid(value){
-        return value === undefined || value === null || value === '' || value === 0;
+        return value === undefined || value === null || value === '' || value === 0 || (_.isArray(value) && value.length === 0) || (_.isObject(value) && _.values(value).length === 0);
```
Comment (mdumrauf) [inline src/EmptyValidator.js]: Can all these be replaced by `_.isEmpty()`?

Comment (Gamuza469) [reply src/EmptyValidator.js]: Not possible. `_.isEmpty() ` considers all numbers as empty.

Comment (Gamuza469) [reply src/EmptyValidator.js]: Should `yarn.lock` be added to `.gitignore` then?

Comment (Gamuza469) [reply src/EmptyValidator.js]: Done both.

**Pull request merged**